build.bash

```diff
@@ -37,14 +28,27 @@ for i in `ls $SCRIPT_DIR/src | grep -v vendor`
 do
 	echo $i
 	go install $i
+	if [[ $? != 0 ]]; then
+	    LINUX_PACKAGES_ERRORS=true
```

Comment:
Intention was to continue building the rest of the packages anyway to see if there are any other issues.